tests/tests/kernel/private/rest/ezprest_request_parser_regression.php

```diff
@@ -83,7 +83,10 @@ public function testRunRegression( $name )
     private static function normalizedVarExport( $var )
     {
         $var = var_export( $var, true );
-        if ( PHP_VERSION_ID < 50430 )
+        // 50505 is PHP 5.5.5
+        // 50509 is PHP 5.5.9
+        // version from 50505 to 50509 are also affected
+        if ( PHP_VERSION_ID <= 50430 || ( PHP_VERSION_ID >= 50505 && PHP_VERSION_ID <= 50509 ) )
```

Comment:
This was fixed in 5.4.30, thus it shouldn't include it, right?

For 5.5, it was apparently fixed in 5.5.14 (http://php.net/ChangeLog-5.php#5.5.14), so why 5.5.9 as the upper range instead of 5.5.14?

And did we confirm that the lower range is 5.5.5? I know you tested on 5.5.5, but do we know whether it's a problem or not on 5.5.0 to 5.5.4?